ruby/example_code/cloudwatch/README.md

```diff
@@ -27,6 +27,27 @@ Shows how to use the AWS SDK for Ruby to work with Amazon CloudWatch.
 For prerequisites, see the [README](../../README.md#Prerequisites) in the `ruby` folder.
 
 
+### Single actions
+Code excerpts that show you how to call individual service functions.
+
+* [Create an alarm](./cw-ruby-example-create-alarm.rb) (`CreateAlarm`)
```

Comment:
These don't look generated by WRITEME, the newlines and missing line numbers from the links would indicate that. Did you have to add these manually for some reason?

Reply:
These I added back because WRITEME doesn't generate them.

Reply:
Writeme should generate anything in the metadata. Are these not SOS'd?

Reply:
Ok, is that the intent or an oversight?

Reply:
These will be added to SOS as part of https://github.com/awsdocs/aws-doc-sdk-examples/issues/4823